extensions/amp-iframe/1.0/storybook/Basic.js

```diff
@@ -1,11 +1,12 @@
 import * as Preact from '#preact';
 import {Iframe} from '../component';
-import {text, withKnobs} from '@storybook/addon-knobs';
```

Comment:
The knobs tab will go away once we remove the addon line in the main.js files right?
<img width="903" alt="Screen Shot 2021-09-02 at 10 04 26 AM" src="https://user-images.githubusercontent.com/10456171/131858064-21453061-0682-4115-aa61-d2487ef6f4d7.png">

Reply:
That's correct.